apps/web/components/booking/BookingListItem.tsx

```diff
@@ -598,9 +598,7 @@ function BookingListItem(booking: BookingItemProps) {
         <td className="flex w-full flex-col flex-wrap items-end justify-end space-x-2 space-y-2 py-4 pl-4 text-right text-sm font-medium ltr:pr-4 rtl:pl-4 sm:flex-row sm:flex-nowrap sm:items-start sm:space-y-0 sm:pl-0">
           {isUpcoming && !isCancelled ? (
             <>
-              {isPending && (userId === booking.user?.id || booking.isUserTeamAdminOrOwner) && (
```

Comment:
Will this show actions for unauthorized users in the UI?

Reply:
Yeah the idea is to show all the actions to user but check for authorization on the backend when user clicks on any action

Reply:
How much of a performance improvement are we seeing doing this?